index.html

```diff
@@ -1,14 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <nav>
+            <a href="">
+                <h4>My Logo</h4>
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
+    </header>
+
+    <main>
+        <section class="Profile">
+            <article>
+                <h1>I’m Temesgen <br> Glad to see you!</h1>
+                <p>
+                    I’m a software developer! I can help you
+                    build a product, feature or website. Look
+                    through some of my work and experirience!
+                    If you like what you see and have a project
+                    you need coded, don't hestitate to contact me.
+                </p>
+            </article>
+            <h4>Lets connect</h4>
+            <ul class="social-handle">
+                <li><a href="#"><img src="./img/twitter.png" alt="twitter icon"></a></li>
```

Comment:
- [ ] sticking to the Figma design as much as you can it's a valuable skill to have as a developer because it's what you will be doing in the future when implementing mockups into real projects.so you should consider positioning the background correctly to match the Figma design.

Your design:

![image](https://user-images.githubusercontent.com/90524466/170819055-778ecbd5-2e4e-4ce1-9b62-be5a4837397a.png)


Figma design:

![image](https://user-images.githubusercontent.com/90524466/170819089-06d9ed88-0875-46f4-962b-f5a706cbb8be.png)